ocs_ci/ocs/utils.py

```diff
@@ -721,6 +721,24 @@ def get_pod_name_by_pattern(
     return pod_list
 
 
+def get_rook_version():
```

Comment:
we have already same function here : https://github.com/red-hat-storage/ocs-ci/blob/1e0d9838cc2cd8f7c8d829531149f208ae3d4a3e/ocs_ci/utility/utils.py#L1493

Reply:
The function in `ocs-ci/ocs_ci/utility/utils.py` itself requires ceph toolbox to be present to execute `rook version` command.
Here, we fetch the image information from rook-ceph-operator pod.